vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -18,6 +18,29 @@ function error {
   local D=`date`
   echo [ ERROR - $D ] $* >&2
 }
+function udocker_function {
+  #installation of udocker
+  git clone --depth=1 https://github.com/indigo-dc/udocker.git
+  (cd udocker/udocker; ln -s maincmd.py udocker)
+  export PATH=`pwd`/udocker/udocker:$PATH
+  #creating a temporary directory for udocker containers
+  mkdir -p containers
+  export UDOCKER_CONTAINERS=$PWD/containers
+  #find pre-deployed containers on CVMFS, and create a symlink to the udocker containers directory
+  for d in /cvmfs/biomed.egi.eu/vip/udocker/containers/*/ ; do mkdir containers/$(basename "${d%/}") && ln -s "${d%/}"/* containers/$(basename "${d%/}")/;done
```

Comment:
The path "/cvmfs/biomed.egi.eu/vip/udocker/containers" should be en env variable (like BOSH_CVMFS_PATH)